Tests/AdyenTests/Adyen Tests/Components/PresentationDelegateMock.swift

```diff
@@ -2,8 +2,15 @@
 
 import Adyen
 import Foundation
+@testable import AdyenDropIn
 
-final class PresentationDelegateMock: PresentationDelegate {
+final class PresentationDelegateMock: NavigationProtocol {
+
+    var doDismiss: ( ((() -> Void)?) -> Void )?
```

Comment:
i think this is getting out of hand 🥲

Reply:
This is just a mock 😅
I need to test, that if I call `didFinalize` - I got `completion` back